index.js

```diff
@@ -42,6 +42,15 @@ function servePage(path, res) {
     });
 }
 
+function validToken(token){
```

Comment:
```suggestion
async function validToken(token){
```
Function needs to be async to have await in it.